e2e/src/setup-drivers.js

```diff
@@ -44,19 +44,19 @@ async function deployDriverContract(api, txqueue, system, pair, cert, contract,
             transfer: u128,
         },
      */
-    const instantiateReturn = await system.instantiate({
+    const instantiateResult = await system.instantiate({
         codeHash: contract.metadata.source.hash,
         salt,
         instantiateData: contract.constructor, // please concat with args if needed
         deposit: 0,
         transfer: 0,
     }, cert);
 
-    console.log("instantiate result:", instantiateReturn);
-    const queryResponse = api.createType('InkResponse', instantiateReturn);
-    const queryResult = queryResponse.result.toHuman()
-    console.log("InkMessageReturn", queryResult.Ok.InkMessageReturn);
-    const instantiateResult = api.createType('ContractInstantiateResult', queryResult.Ok.InkMessageReturn);
+    // console.log("instantiate result:", instantiateReturn);
```

Comment:
What's the comment about?

Reply:
Dead codes, relative to this commit: https://github.com/Phala-Network/phala-blockchain/commit/ac1b0ade95a8bb9b11f474184569189e0917eb63

Which move the logic into SDK.

Reply:
I'm not sure this script still in use or not, it's broken as I checked.